dlp/snippets/src/test/java/dlp/snippets/InfoTypesTests.java

```diff
@@ -38,4 +40,17 @@ public void testListInfoTypes() throws Exception {
     assertThat(output).contains("Name");
     assertThat(output).contains("Display name");
   }
+
+  @Test
+  public void testCreateStoredInfoType() throws IOException {
+    CreateStoredInfoType.createStoredInfoType(PROJECT_ID, GCS_PATH);
+    String output = bout.toString();
+    String storedInfoTypeId = output.split("Created Stored InfoType: ")[1].split("\n")[0];
+    assertThat(output).contains("Created Stored InfoType: ");
```

Comment:
This assertion should probably go before the split, otherwise you're implicitly asserting that the output contained this string (split(...)[1] won't exist if the output did not have the splitter)